app/models/item.rb

```diff
@@ -1,5 +1,18 @@
 class Item < ActiveRecord::Base
   belongs_to :list
+  acts_as_list scope: 'list_id = #{list_id} AND done = false'
 
   validates :text, :list_id, presence: true
+
+  def mark
+    Item.transaction do
+      update_attributes done: !done
```

Comment:
@vlasiak, Here you can simply use `transaction` method of the instance (no need to do it thought `Item` class).